.circleci/config.yml

```diff
@@ -179,6 +179,42 @@ jobs:
   #     - store_test_results:
   #         path: /go/out/tests
 
+  e2e-simple-distroless:
```

Comment:
we are migrating off of circle (planning on fully turning it down in days, not just "eventually"), so I think it would be best to have this run on prow. You can add a script to the `prow/` folder, probably almost identical to the existing e2e-simple one.

Then to turn on the test you can open a PR in the istio/test-infra folder

Reply:
@howardjohn, we migrated the tests to prow now.